### PR TITLE
Wishlist added

### PR DESCRIPTION
**Pull request by vanshitamoolchandani**
image

### CODE REVIEW

**Review by Anurag021 — APPROVED** [review]
Code looks good to me, UI is great too.

File: Pages/Wishlist/styles.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+@import url('/Pages/Product-Listing-Page/styles.css');
+
+.component-heading{
+    text-align: center;
+}
+.main-wishlist{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 22rem;
+    /* grid-auto-flow: column; */
```
Comment (sri-rishi) [inline Pages/Wishlist/styles.css]: Just remove this comment

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Otherwise code  and UI looks good to me

**Review by Sahil25061999 — APPROVED** [review]
didnt find anything just a suggestion for navbar icons you can add cursor: pointer

**Pull request merged**